extras/systemd/dynflow-sidekiq@.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Foreman jobs daemon on sidekiq
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=forking
+User=foreman
+TimeoutSec=90
```

Comment:
We should likely make this configurable

Reply:
Isn't the drop-in configuration for the service (i.e. overrides from `/etc/systemd/system/dynflow-sidekiq@.service.d/10-timeout_60s.conf`) enough?

Reply:
If we configure the puppet module to allow deploying this override file or straight up configuring the timeout directly then I think we are covered.

Reply:
We'll probably go with systemd drop ins and the installer should configure those. Marking as resolved.